xml/System.Diagnostics.Tracing/DiagnosticCounter.xml

```diff
@@ -89,7 +89,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Removes the counter from set that the EventSource will report on.  After being disposed, this counter will do nothing and its resource will be reclaimed if all references to it are removed. If an EventCounter is not explicitly disposed it will be cleaned up automatically when the EventSource it is attached to dies.</summary>
```

Comment:
```suggestion
        <summary>Removes the counter from set that the EventSource will report on.  After being disposed, this counter will do nothing and its resources will be reclaimed if all references to it are removed. If an EventCounter is not explicitly disposed it will be cleaned up automatically when the EventSource it is attached to dies.</summary>
```

Reply:
Thanks. Adding a few missing crefs:

```suggestion
        <summary>Removes the counter from set that the <see cref="T:System.Diagnostics.Tracing.EventSource" /> will report on. After being disposed, this counter will do nothing and its resources will be reclaimed if all references to it are removed. If an <see cref="T:System.Diagnostics.Tracing.EventCounter" /> is not explicitly disposed, it will be cleaned up automatically when the <see cref="T:System.Diagnostics.Tracing.EventSource" /> it is attached to dies.</summary>
```